bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,452 @@
+        """Ensures that each user adds one and only one reaction."""
+        async with self.checking_messages:  # Acquire the lock so that the dictionary isn't reset while iterating.
+            if reaction.emoji in EMOJIS_VAL and await self.messages.contains(
+                reaction.message.id
+            ):
+                # create a custom counter
+                reaction_counter = defaultdict(int)
+                for reaction in reaction.message.reactions:
+                    async for reaction_user in reaction.users():
+                        if reaction_user == self.bot.user:
+                            continue
+                        reaction_counter[reaction_user] += 1
+
+                if reaction_counter[user] > 1 and user != self.bot.user:
+                    await user.send(
+                        "Sorry, you have already added a reaction, "
+                        "please remove your reaction and try again."
+                    )
+                    return await reaction.remove(user)
+
+    @spook_name_rate.command(name="delete")
+    async def delete_name(self, ctx: Context) -> None:
+        """Delete the user's name."""
+        if self.poll:
+            await ctx.send(
+                "You can't delete your name since the poll has already started!"
+            )
+            return
+        for message_id, data in await self.messages.items():
+            data = json.loads(data)
+
+            if ctx.author.id == data["author"]:
+                await self.messages.delete(message_id)
+                await ctx.send(f'Message deleted successfully ({data["name"]!r})!')
+                return
+
+        await ctx.send(
+            "But you don't have an entry... :eyes: Type `spooknamerate add your entry`"
+        )
+
+    @tasks.loop(hours=24.0)
+    async def announce_name(self) -> None:
+        """Announces the name needed to spookify every 24 hours and the winner of the previous game."""
+        if not self.in_allowed_month():
+            return
+
+        channel = await self.get_channel()
+
+        if self.first_time:
+            await channel.send(
+                "Okkey... Welcome to the **Spook Name Rate Game**! It's a relatively simple game.\n"
+                f"Everyday, a random name will be sent in <#{Channels.community_bot_commands}> "
+                "and you need to try and spookify it!\nRegister your name using "
+                f"`{self.bot.command_prefix}spooknamerate add spookified name`"
+            )
+
+            await self.data.set("first_time", False)
+            self.first_time = False
+
+        else:
+            if await self.messages.items():
+                await channel.send(embed=await self.get_responses_list(final=True))
+                self.poll = True
+                if not self.debug:
+                    await asyncio.sleep(
+                        2 * 60 * 60
+                    )  # sleep for 2 * 60 * 60 seconds (two hours)
```

Comment:
Here is no need to make this call 3 lines. Also 2 * 60 * 60 mentioning in comment is also not required, better should be `# sleep for two hours`.

Reply:
Oops I think I had formatted it with black by mistake once, so maybe thats why it spans across 3 lines